instrumentation/graphql-java-12.0/library/src/main/java/io/opentelemetry/instrumentation/graphql/GraphQLTracing.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.graphql;
+
+import graphql.ExecutionResult;
+import graphql.execution.instrumentation.Instrumentation;
+import graphql.execution.instrumentation.parameters.InstrumentationExecutionParameters;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.trace.StatusCode;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+import io.opentelemetry.instrumentation.api.instrumenter.InstrumenterBuilder;
+import io.opentelemetry.instrumentation.api.instrumenter.SpanStatusExtractor;
+
+@SuppressWarnings("AbbreviationAsWordInName")
```

Comment:
as discussed on slack, for library instrumentation I used `GraphQL` which isn't allowed by our checkstyle rules